packages/genesys-spark-components/src/style/style.scss

```diff
@@ -2,8 +2,7 @@
  * This file combines all the exported library styles into a single stylesheet for consumption by applications.
  */
 
-@use '~genesys-spark-tokens/dist/css/gse-core.css';
-@use '~genesys-spark-tokens/dist/css/gse-semantic.css';
+@use '~genesys-spark/src/style/global.scss';
```

Comment:
This is why this is not a breaking change right? The stylesheet exported with the components will be unchanged (except maybe for the internal order that is not something we depended on)

Reply:
Correct. In theory, the existing stylesheet should be more or less unchanged. This just allows for people to import the global styles (and the new reset styles) separately - even without using the component.